cli/bpmetadata/cmd.go

```diff
@@ -233,23 +231,23 @@ func createInfo(bpPath string, readmeContent []byte) (*BlueprintInfo, error) {
 		i.Icon = iconFilePath
 	}
 
-	return i, nil
+	return nil
 }
 
-func createInterfaces(bpPath string, interfaces *BlueprintInterface) (*BlueprintInterface, error) {
+func (i *BlueprintInterface) create(bpPath string) error {
 	i, err := getBlueprintInterfaces(bpPath)
 	if err != nil {
-		return nil, err
+		return err
 	}
 
-	if interfaces.VariableGroups != nil {
-		i.VariableGroups = interfaces.VariableGroups
+	if i.VariableGroups != nil {
+		i.VariableGroups = i.VariableGroups
```

Comment:
I believe this is a noop?

Reply:
Yeah, this won't apply since it's already unmarshaled.